### ISSUE TITLE
Failed to clone on Arch lInux

### ISSUE THREAD

**Issue by kostya** [crystal-lang/shards#497]
```
Crystal 1.0.0 [dd40a2442] (2021-03-22)
Shards 0.14.1 [addc26a] (2021-03-10)
Arch Linux
```

```
dependencies:
  clear:
    github: anykeyh/clear
    branch: master
```

`shards update`
```
shards update
Resolving dependencies
Fetching https://github.com/anykeyh/clear.git
Fetching https://github.com/jwaldrip/admiral.cr.git
Fetching https://github.com/anykeyh/generate.cr.git
Fetching https://github.com/will/crystal-pg.git
Fetching https://github.com/anykeyh/inflector.cr.git
Failed to clone https://github.com/anykeyh/inflector.cr.git
```

but `git clone https://github.com/anykeyh/inflector.cr.git` locally work.

**Comment by kostya** [comment]
after `rm -rf ~/.cache/shards/`, now every shard failed to clone, not understand whats wrong, git clone works.

**Comment by kostya** [comment]
btw on Ubuntu all works ok. But not on Arch, i think may be this is related to dns-resolv issues (because i use custom /etc/resolv.conf).

**Comment by straight-shoota** [comment]
Could you run with `--verbose` flag and show the output?

**Comment by kostya** [comment]
I tried with --verbose and run command by hands:
```
git clone -c core.askPass=true --mirror --quiet -- https://github.com/kostya/myhtml.git myhtml
Segmentation fault (core dumped)
```

i find that remove `--mirror` fix this segfault. Git 2.31.1. So this is looks like Arch package bug, or some related packages bug, like openssl, or something.

**Comment by kostya** [comment]
This was git bug, and would be fixed in next after 2.31.1 version.
